### PR TITLE
[MOSIP-29895] Release 1.2.0.1 mosip 29895 1

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**

### CODE REVIEW

File: registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java [hunk 1]
```diff
@@ -154,6 +158,8 @@ public class PacketValidateProcessor {
 	private static final String PRE_REG_ID = "mosip.pre-registration.datasync.store";
 	private static final String VERSION = "1.0";
 
+	String registrationId = null;
```
Comment (sowmya695) [inline registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java]: while resolving the conflicts you have copied to above ,this is the wrong place its should be within process

File: registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java [hunk 2]
```diff
@@ -166,12 +172,13 @@ public class PacketValidateProcessor {
...
 	private NotificationUtility notificationUtility;
 
+	@Value("${mosip.registration.processor.datetime.pattern}")
+	private String dateformat;
+
 	public MessageDTO process(MessageDTO object, String stageName) {
 		TrimExceptionMessage trimMessage = new TrimExceptionMessage();
 		LogDescription description = new LogDescription();
 		PacketValidationDto packetValidationDto = new PacketValidationDto();
-		String registrationId = null;
```
Comment (sowmya695) [inline registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java:173]: it should be add here

File: registration-processor/pre-processor/registration-processor-packet-validator-stage/src/test/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessorTest.java [hunk 3]
```diff
@@ -245,43 +249,64 @@ public void setup() throws Exception {
...
-		assertFalse(object.getInternalError());
+		ArgumentCaptor<InternalRegistrationStatusDto> argument = ArgumentCaptor
+				.forClass(InternalRegistrationStatusDto.class);
+		Mockito.verify(registrationStatusService,Mockito.atLeastOnce()).updateRegistrationStatus(argument.capture(),any(),any());
+		assertEquals(null, argument.getAllValues().get(0).getPacketCreateDateTime());
 	}
-	
+
 	@Test
-	public void invalidSupervisorStatusTest() throws PacketValidatorException {
```
Comment (sowmya695) [inline registration-processor/pre-processor/registration-processor-packet-validator-stage/src/test/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessorTest.java:275]: Next time dont override the test cases ,please add the new test cases at the end of class

**Review by sowmya695 — CHANGES_REQUESTED** [review]
Please fix the review comment

File: registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java (hunk 1 above)
Comment (Khuddusshariff0022) [reply registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java]: done

File: registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java (hunk 2 above)
Comment (Khuddusshariff0022) [reply registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java:173]: added here

**Review by sowmya695 — APPROVED** [review]

**Review by vishwa-vyom — APPROVED** [review]
Approving based on Sowmya's review

**Pull request merged**